sdcm/nemesis.py

```diff
@@ -6804,3 +6847,21 @@ class IsolateNodeWithIptableRuleNemesis(Nemesis):
 
     def disrupt(self):
         self.disrupt_refuse_connection_with_block_scylla_ports_on_banned_node()
+
+
+class ManagerRcloneBackup(Nemesis):
+    manager_operation = True
+    disruptive = False
+    supports_high_disk_utilization = False
+
+    def disrupt(self):
+        self.disrupt_manager_backup(object_storage_upload_mode=ObjectStorageUploadMode.RCLONE, label='rclone_backup')
+
+
+class ManagerNativeBackup(Nemesis):
+    manager_operation = True
+    disruptive = False
+    supports_high_disk_utilization = False
+
+    def disrupt(self):
+        self.disrupt_manager_backup(object_storage_upload_mode=ObjectStorageUploadMode.NATIVE, label='native_backup')
```

Comment:
this will never be executed, the nemesis will be run, `self.disrupt_manager_backup()` will be executed instead

Reply:
It would be selected but won't use the code mentioned above.

If there is a reason you do want this nemesis run in all cases, it should be documented within the code

And if it's needed to ignore by default it should be added to the complex nemesis list to be excluded explicitly,  now like that, since the code might be refact at some point and it would start running

Reply:
@pehala , fixed that in a new commit - added nemesis to excluded list.